packages/flame_isolate/lib/flame_isolate.dart

```diff
@@ -50,12 +51,41 @@ mixin FlameIsolate on Component implements IsolateGetter {
     _isolate = null;
   }
 
-  @override
+  /// Deprecated in favor of [isolateCompute].
+  @Deprecated('Use isolateCompute instead.')
```

Comment:
Write which version it will be removed in, v0.6.0 I guess?

Reply:
Well, it is already removed in the underlying package, so this is a soft deprecation that can be removed in the flame_isolate package whenever.

Hence a bit up to you and the best practices of Flame :)

Reply:
v0.7.0 then, since this one will be 0.6.0 now due to the `feat` :)